### DISCUSSION TITLE
Adding Attribute

### DISCUSSION THREAD

**Discussion by grov** [dbarzin/deming#35] [General]
Hello,

First of all, thanks for this great tool !
I have an issue with the attribute functionnality. 
When i try to add one i get the error message "The values format is invalid.".
Here is what i wrote : 
Name : Account review
Values : Account_Review (or #Account_Review) 

what am i doing wrong? :)
Thanks ! 
Yoann

**Comment by dbarzin** [comment]
Could not reproduce the issue :
![image](https://github.com/dbarzin/deming/assets/43208232/f7eac76d-1f53-4697-8de4-ae5b2d0b32f6)
Could you update your application to the last version ?
Have you followed the installation procedure ?
Is there something specific in your environment ?

**Reply by grov** [reply]
Hello ! Thanks for answering ! 
I used the Docker compose installation method. 
Here is a video of the manipulation i did :  https://youtu.be/N9xpQZy1PTI

Thanks !
grov

**Comment by dbarzin** [comment]
It has been fixed.
Thanks,
Didier
